its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/OrchestratorState.java

```diff
@@ -47,6 +48,17 @@ public void startOnce() throws Exception {
       usageCount += 1;
       if (usageCount == 1) {
         orchestrator.start();
+        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfile.xml"));
+        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileCpp.xml"));
+        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileExternalIssuesVB.xml"));
+        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileExternalIssues.xml"));
```

Comment:
Could be forwarded to the cleanup checklist
```suggestion
        orchestrator.getServer().restoreProfile(FileLocation.of("qualityProfiles/TestQualityProfileExternalIssuesCS.xml"));
```